src/main/webapp/script.js

```diff
@@ -104,20 +104,36 @@ function createParagraphElement(text) {
 // This function returns a message based on sentiment scores
 function getScoreResponse(score) {
   // Messages based on score ranges
-  //TODO: update messages to be something more readable
-  const messages = ["Negative", "Neutral", "Positive"];
+  //TODO: updated messages to a series of good/bad/netural messages. Make it a random selection.
+  const positiveMessages = [
+    "Liking the positive energy!!",
```

Comment:
Not fully committed to these responses. Let me know if you have additional messages to add or to change certain ones.